ladybug/epw.py

```diff
@@ -205,19 +205,21 @@ def _import_data(self, import_location_only=False):
                             raise ValueError(e)
                         value = int(round(float(data[field_number])))
 
-                    if start_time == 0.5:
+                    if middle_hour is True:
                         self._data[field_number].append(
-                            DataPoint(value, timestamp_half_hour))
+                            DataPoint(value, timestamp_middle_hour))
                     else:
-                        if insert_hour is False:
-                            self._data[field_number].append(
-                                DataPoint(value, timestamp_hour))
-                        else:
-                            self._data[field_number].insert(
-                                0, DataPoint(value, DateTime(1, 1, 0)))
+                        self._data[field_number].append(
+                            DataPoint(value, timestamp_hour))
 
                 line = epwin.readline()
 
+            # move last item to start position for fields on the hour
+            for field_number in xrange(self._num_of_fields):
+                middle_hour = EPWFields.field_by_number(field_number).middle_hour
+                if middle_hour is False:
+                    self._data[field_number].insert(0, self._data[field_number].pop())
```

Comment:
I suggest to write this in two separate lines so it's more clear what's happening.
```python
last_item = self._data[field_number].pop()
self._data[field_number].insert(0, last_item)
```

Reply:
ok